pkgs/by-name/al/alphafold3/package.nix

```diff
@@ -0,0 +1,118 @@
+{
+  lib,
+  fetchFromGitHub,
+  replaceVars,
+  python3,
+  abseil-cpp,
+  dssp,
+  libcifpp,
+  pybind11-abseil,
+  zlib,
+  hmmer,
+}:
+
+let
+  python = python3.override {
+    self = python;
+    packageOverrides = final: prev: {
+      typeguard = prev.typeguard.overridePythonAttrs (oldAttrs: rec {
+        version = "2.13.3";
+        src = oldAttrs.src.override {
+          inherit version;
+          hash = "sha256-AO2qjaOhM2dHls9eqH2fS0w2fXdHbhhegCUcwT37uMQ=";
+        };
+        doCheck = false;
+      });
+    };
+  };
+  libcifpp' = libcifpp.override { downloadCCDFile = true; };
+in
+python.pkgs.buildPythonApplication {
+  pname = "alphafold3";
+  version = "3.0.0-unstable-2024-12-20";
```

Comment:
Is there any issue upstream that it requires a intermediary commit?

Reply:
[90 commits](https://github.com/google-deepmind/alphafold3/compare/v3.0.0...b380a7c085a5eeea5117d2bd43c121e9a6e0df64) have been added.
I don't expect any new releases since deepmind is developing it.